.sbtopts

```diff
@@ -1,4 +1,3 @@
 -J-Xms2048m
 -J-Xmx2048m
 -J-XX:ReservedCodeCacheSize=256m
--J-XX:MaxMetaspaceSize=512M
```

Comment:
This line is dropped after seeing this in the build log:

```log
[error] Caught java.lang.OutOfMemoryError: Metaspace
[error] To best utilize classloader caching and to prevent file handle leaks, we recommend running sbt without a MaxMetaspaceSize limit.
[error]
java.lang.OutOfMemoryError: Metaspace
```